### DISCUSSION TITLE
Issue with machine-config on 4.14

### DISCUSSION THREAD

**Discussion by marekk1717** [okd-project/okd#2011] [General]
I've got a problem with single node installation of 4.14.0-0.okd-2024-01-26-175629

I've tried already few times from scratch.

Machine config operator throws this error:
machine-config                             4.14.0-0.okd-2024-01-26-175629   True        False         True       24h     Failed to resync 4.14.0-0.okd-2024-01-26-175629 because: error during syncRequiredMachineConfigPools: context deadline exceeded

Machine config deamon pod throws this:
I0516 13:17:33.275245 8365 daemon.go:1370] In bootstrap mode
E0516 13:17:33.275335 8365 writer.go:226] Marking Degraded due to: machineconfig.machineconfiguration.openshift.io "render-worker-db56bc37dd86657823ec373bb5904b67" not found

oc get mcp
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master                                                      False     True       True       1              0                   0                     1                      11m
worker   rendered-worker-632f2191dd1fb7ed02d61df84649d5a3   True      False      False      0              0                   0                     0                      24h

oc get mc
NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
00-master                                          02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
00-worker                                          02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
01-master-container-runtime                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
01-master-kubelet                                  02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
01-worker-container-runtime                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
01-worker-kubelet                                  02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
97-master-generated-kubelet                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
97-worker-generated-kubelet                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
98-master-generated-kubelet                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
98-worker-generated-kubelet                        02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
99-master-generated-registries                     02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
99-master-ssh                                                                                 3.2.0             24h
99-okd-master-disable-mitigations                                                             3.2.0             24h
99-okd-worker-disable-mitigations                                                             3.2.0             24h
99-worker-generated-registries                     02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
99-worker-ssh                                                                                 3.2.0             24h
rendered-master-3e8239ad26f6199b575759c8d2d923c0   02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             12m
rendered-worker-5534c0d4d3c29a786da6495d5c67afb6   02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             24h
rendered-worker-632f2191dd1fb7ed02d61df84649d5a3   02a6260d730bae5995e3bde6c6792a75db9ec025   3.4.0             38m

**Comment by JaimeMagiera** [comment]
Hi,

We are not working on FCOS builds of OKD any more. Please see these documents...

https://okd.io/blog/2024/06/01/okd-future-statement
https://okd.io/blog/2024/07/30/okd-pre-release-testing

We will be providing documentation on upgrading clusters from 4.15 FCOS to 4.16 SCOS. In terms of clusters that are older, you may be able to get help from community members. I'll convert this to a discussion to facilitate that.

Many thanks,

Jaime
